docs/userguide/manage_files.rst

```diff
@@ -0,0 +1,177 @@
+.. Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+.. _doc_userguide_manage_files:
+
+Working with files and folders
+==============================
+
+You can import files into a workspace. Files in a workspace are automatically shared with the users who have access to the workspace (see :ref:`Share a workspace <doc_userguide_parsec_workspaces_share>`.). Depending on your :ref:`workspace role <doc_userguide_parsec_workspaces_roles>`, some features may not be available.
+
+
+Importing files
+---------------
+
+Using the import button
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Click on the button ``Import`` in the action bar and select ``Import files`` or ```Import a folder``.  Select the files or folder to import from your file system. The files will be imported in the current directory.
+
+.. image:: screens/files/import_buttons.png
+    :align: center
+    :alt: import buttons screenshot
+    :width: 400
+
+Using drag and drop
+^^^^^^^^^^^^^^^^^^^
+
+You can drag and drop files and folders directly from your file explorer to Parsec. This is the preferred method as you are able to import both files and folders at the same time. You can either drop the files on an empty space to import them in the current folder, or on a specific folder to place them inside it.
+
+.. image:: screens/files/drag_and_drop.png
+    :align: center
+    :alt: drag and drop screenshot
+
+The import window
+^^^^^^^^^^^^^^^^^
+
+When you import files, the import windows appears automatically to show the progress. You can use Parsec during the import process. We strongly recommend that you do not close Parsec or log out before it has finished to guarantee that all your files are being imported.
+
+.. image:: screens/files/import_window.png
+    :align: center
+    :alt: import window screenshot
+    :width: 400
+
+When a file is imported, Parsec will start synchronizing it right away with the server. The cloud icon next to the file name indicates its synchronization status:
+
+* File/Folder has not yet been synchronized yet.
+
+  .. image:: screens/files/not_synced.png
+    :width: 64
+
+
+* File/Folder has been fully synchronized with the server and is available to other users in the workspace.
+
+  .. image:: screens/files/synced.png
+    :width: 64
+
+
+Managing files and folders
+--------------------------
```

Comment:
Maybe it is better to avoid "managing" here :thinking: 

We could call this section "Working with files and folders" and rename the main title (at the top of the file) to just "Files and folders". I'm not 100% sure though, what do you think?

```suggestion
Working with files and folders
------------------------------
```